src/promptflow/promptflow/_sdk/operations/_run_operations.py

```diff
@@ -140,7 +140,10 @@ def stream(self, name: Union[str, Run]) -> Run:
             available_logs = local_storage.logger.get_logs()
             incremental_print(available_logs, printed, file_handler)
             self._print_run_summary(run)
-            # won't print error here, put it in run dict
+            # print error message when run is failed
+            if run.status == RunStatus.FAILED:
+                error_message = local_storage.load_exception()["message"]
+                file_handler.write(f"{error_message}\n")
```

Comment:
maybe we should write this as read in cli?

Reply:
updated to use red print for better visibility